play-v27/src/test/scala/com/gu/googleauth/AntiForgeryCheckerTest.scala

```diff
@@ -18,7 +18,9 @@ class AntiForgeryCheckerTest extends AnyFlatSpec with Matchers with TryValues {
 
   val ExampleSessionId = AntiForgeryChecker.generateSessionId()
 
-  val antiForgery = AntiForgeryChecker(InitialSecret("reallySecret"), HS256)
+  def lengthySecret(prefix: String) = s"${prefix}_${"pad" * 256}"
+
+  val antiForgery = AntiForgeryChecker(InitialSecret(lengthySecret("reallySecret")), HS256)
```

Comment:
The newer versions of Play (v2.9 and later) pull in a newer version of [`JJWT`](https://github.com/jwtk/jjwt), which (due to https://github.com/jwtk/jjwt/issues/334) **started enforcing key length** as of JJWT v0.10.0.

In the `AntiForgeryCheckerTest`, we weren't using keys of sufficient length to avoid this enforcement:


> [info] AntiForgeryCheckerTest:
> [info] Anti Forgery
> [info] - should fail if token is signed with other algorithm, even if it has the same secret *** FAILED ***
> [info]   io.jsonwebtoken.security.WeakKeyException: The signing key's size is 152 bits which is not secure enough for the HS384 algorithm.  The JWT JWA Specification (RFC 7518, Section 3.2) states that keys used with HS384 MUST have a size >= 384 bits (the key size must be greater than or equal to the hash output size).  Consider using the io.jsonwebtoken.security.Keys class's 'secretKeyFor(SignatureAlgorithm.HS384)' method to create a key guaranteed to be secure enough for HS384.  See https://tools.ietf.org/html/rfc7518#section-3.2 for more information.


...so we need `lengthySecret()` keys!